Uplift/Views/GymDetail/Facilities/MiscellaneousInfoView.swift

```diff
@@ -8,29 +8,32 @@
 
 import UIKit
 
-class MiscellaneousInfoView: UIView {
+class MiscellaneousInfoView: UICollectionViewCell {
 
     private let textView = UITextView()
 
-    init(miscellaneousInfo: [String]) {
+    init() {
         super.init(frame: CGRect.zero)
```

Comment:
we should be overriding the default init here:
`override init(frame: CGRect) {...}`